doc/installation.rst

```diff
@@ -80,11 +80,13 @@ computer, you can install them in your home directory like this:
 ::
 
   cd <dir>
+  rm -rf build Build
```

Comment:
I never had a problem reusing the Build directory, but I did have some problems with build.  So ok, I'll merge it like this.